README.md

```diff
@@ -13,10 +13,9 @@ This server requires a working copies of Node.js, Yarn, PM2, and CouchDB. We als
 Run these commands as a server admin:
 
 ```sh
-touch /var/log/pushServer.log
-touch /var/log/priceDaemon.log
```

Comment:
We don't need to run these `touch` commands anymore? If not, it LTGM.

Reply:
Since the folder belongs to the `edgy` user the app itself can create the files now.